src/Fixer/FunctionNotation/FunctionDeclarationFixer.php

```diff
@@ -229,6 +231,10 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 ->setDefault(self::SPACING_ONE)
                 ->setAllowedValues(self::SUPPORTED_SPACINGS)
                 ->getOption(),
+            (new FixerOptionBuilder('closure_fn_spacing', 'Spacing to use before open parenthesis for short arrow functions.'))
+                ->setDefault(self::SPACING_ONE)
```

Comment:
maybe sth like https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Fixer/ControlStructure/NoAlternativeSyntaxFixer.php#L78 ?

Reply:
I added a TODO comment for this.